.idea/codeStyles/Project.xml

```diff
@@ -1,7 +1,7 @@
 <component name="ProjectCodeStyleConfiguration">
```

Comment:
Please revert these style changes, or send out a dedicated PR to change them so other team members can comment on them.

Reply:
Done